x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -860,7 +860,6 @@ type KbnConfigSchemaNonOptionalProps<Props extends Record<string, unknown>> = Pi
  */
 export interface PolicyConfig {
   windows: {
-    advanced?: AdvancedFields; // will be introduced in 7.11+
```

Comment:
For the type failure - you can actually keep this, but have it accept any object since the advanced section can dynamically accept a set of integrations.

Something like this should work:

```js
...
windows {
   advanced?: {};
...
}
...
```

@paul-tavares @parkiino let me know your thoughts

Reply:
yeah that makes sense to me